content/en/docs/getting-started/_index.md

```diff
@@ -17,38 +17,37 @@ These pages are under construction. TODO: Check Prerequisites and update
 
 - [git](https://git-scm.com/downloads)
 - [go](https://golang.org/dl/) version `v1.12+`.
-- [docker](https://docs.docker.com/install/) version `17.03`+.
-  - Alternatively [podman](https://github.com/containers/libpod/blob/master/install.md) `v1.2.0+` or [buildah](https://github.com/containers/buildah/blob/master/install.md) `v1.7+`
+- [docker](https://docs.docker.com/install/) version `17.03`+ or [podman](https://github.com/containers/libpod/blob/master/install.md) `v1.2.0+` or [buildah](https://github.com/containers/buildah/blob/master/install.md) `v1.7+`.
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) `v1.11.3+`.
 - Access to a Kubernetes `v1.11.3+` cluster.
 
 ## Installing OLM in your cluster
 
 ### Install Released OLM
-For installing release versions of OLM, for example version 0.12.0, you can use the following command:
+For installing release versions of OLM, for example version 0.15.1, you can use the following command:
 
-```bash
-export olm_release=0.15.1
-kubectl apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/crds.yaml
-kubectl apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/olm.yaml
+```sh
```

Comment:
If we expect users to copy these shell commands verbatim then I suggest we do not prepend `$ ` to each line of a `sh` block; this prefix is only visually useful when using a `console` block with command output as an example.

Reply:
That makes sense, I prepended the `$ ` prefix to match the style on the SDK website, but I have corrected it.